components/HomePage/12Press/EachPress.jsx

```diff
@@ -1,10 +1,12 @@
 import React from 'react';
-import PropTypes from 'prop-types';
+import Image from 'next/image';
+import { Col, Typography } from 'antd';
 import get from 'lodash/get';
+import PropTypes from 'prop-types';
 import Button from 'common-util/Button';
 import Tag from 'common-util/Tag';
-import { Col } from 'antd';
-import Title from 'antd/lib/typography/Title';
```

Comment:
Is this not more efficient in terms of the code that finally gets sent down the wire?